src/containers/General/Main.tsx

```diff
@@ -1,13 +1,44 @@
 import React from 'react'
+import Voice from 'react-native-voice'
 
-import Screen from '../../screens/General/Main'
+import { Button } from 'react-native'
 
 export default class MainContainer extends React.Component {
+  constructor(props: object) {
+    super(props)
+    Voice.onSpeechResults = this.onSpeechResultsHandler.bind(this)
+  }
+
+  componentWillUnmount() {
+    Voice.destroy().then(Voice.removeAllListeners)
+  }
+
   render() {
-    return <Screen />
+    return (
+      <>
+        <Button onPress={this.onStartButtonPress} title="Start recognize" />
+        <Button onPress={this.onStopPress} title="Stop recognize" />
+      </>
+    )
+  }
+
+  private async onStartButtonPress() {
+    try {
+      await Voice.start('en-US')
+    } catch (e) {
+      console.log(e, 'onStartButtonPressErr')
+    }
+  }
+
+  private async onStopPress() {
+    try {
+      await Voice.stop()
+    } catch (e) {
+      console.log(e, 'onStopPressErr')
+    }
   }
 
-  private testFunc = () => {
-    //
+  private onSpeechResultsHandler = (event: object) => {
+    console.log(event, 'onSpeechResultsHandler')
```

Comment:
This is not enough. Make use of this in the app.

Reply:
I've added minimal functionality for testing